oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexWriter.java

```diff
@@ -158,24 +159,23 @@ private void saveMetrics() {
     }
 
     private void provisionIndex() throws IOException {
-        ElasticsearchIndicesClient client = elasticConnection.getClient().indices();
+        final ElasticsearchIndicesClient esClient = elasticConnection.getClient().indices();
         // check if index already exists
-        if(client.exists(i -> i.index(indexName)).value()) {
+        if (esClient.exists(i -> i.index(indexName)).value()) {
             LOG.info("Index {} already exists. Skip index provision", indexName);
             return;
         }
 
-        // create the new index
         final CreateIndexRequest request = ElasticIndexHelper.createIndexRequest(indexName, indexDefinition);
+        if (LOG.isDebugEnabled()) {
+            StringBuilder sb = new StringBuilder();
+            JsonpUtils.toString(request,sb);
```

Comment:
minor, missing space
```suggestion
            JsonpUtils.toString(request, sb);
```